timely/src/dataflow/operators/core/to_stream.rs

```diff
@@ -55,3 +60,31 @@ impl<C: SizableContainer, I: IntoIterator+'static> ToStream<C> for I where C: Pu
         })
     }
 }
+
+/// Converts to a timely [StreamCore]. Equivalent to [`ToStreamBuilder`] but
+/// uses a [`CapacityContainerBuilder`].
+pub trait ToStream<C: Container> {
```

Comment:
It feels weird that we allow folks to have opinions about the containers used without having opinions about how those containers are populated. Like "use a `Vec` please" but no opinions on what the vec should contain.